doc/users/interactive.rst

```diff
@@ -9,40 +9,41 @@
 .. toctree::
 
 
-When working with data it is often invaluable to be able to interact
-with your plots. In many cases the built in pan/zoom and mouse-location
-tools are sufficient, but you can also use the Matplotlib event system
-to build customized data exploration tools.
+When working with data, interactivity can be invaluable. The pan/zoom and 
+mouse-location tools built into the Matplotlib GUI windows are often sufficient, but 
+you can also use the event system to build customized data exploration tools.
 
 Matplotlib ships with :ref:`backends <what-is-a-backend>` binding to
 several GUI toolkits (Qt, Tk, Wx, GTK, macOS, JavaScript) and third party
 packages provide bindings to `kivy
 <https://github.com/kivy-garden/garden.matplotlib>`__ and `Jupyter Lab
-<https://github.com/matplotlib/ipympl>`__.  For the figures to be
-"live" the GUI event loop will need to be integrated with your prompt. The
-simplest way is to use IPython (see :ref:`below <ipython-pylab>`).
+<https://github.com/matplotlib/ipympl>`__.  For the figures to be interactive, 
```

Comment:
I use "live" here to try and disambiguate it from the several other meanings of the word "interactive".

Reply:
"live" needs to be defined on first usage - I try to do it a little later but I can pull it up here.